webview/src/shared/components/sectionContainer/InfoTooltip.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import cx from 'classnames'
+import { PlotsSection } from 'dvc/src/plots/webview/contract'
+import { SetupSection } from 'dvc/src/setup/webview/contract'
+import styles from './styles.module.scss'
+import { SectionDescription } from './SectionContainer'
+import Tooltip from '../tooltip/Tooltip'
+import { Icon } from '../Icon'
+import { Info, PassFilled, Error } from '../icons'
+
+export enum TooltipIconType {
+  PASSED = 'icon-pass-filled',
+  INFO = 'icon-info',
+  ERROR = 'icon-error'
```

Comment:
[C] values probably don't need the `icon` prefix.

Reply:
Why not have the `TooltipIconType` match the icon directly? You'll be able to directly use it in `getIcon` for the `icon` prop.

Reply:
For the name comparison, better yet: `icon.name === TooltipIconType.INFO.name`.

Reply:
After testing, I realized that enums can only hold strings or numbers. I'm not seeing another way to mimic what an enum does with component values without just letting `InfoTooltip` just take in any icon which I'd like to avoid. Will keep the icons strings for now!

Reply:
You'll need to use a const which is practically the same thing.

Reply:
> You'll need to use a const which is practically the same thing.

Apologies, I can't seem to understand how we could use a const. Could you give me a code example? Here's what I tried:

```
const IconTypes = {
  ERROR: Error,
  PASSED: PassFilled
} as const

type Type = (typeof IconTypes)[keyof typeof IconTypes]

const getNeededIcon = (icon: Type) => {
  // get the icon here
}

// yes this works
getNeededIcon(IconType.PASSED)

// but this also works 
// since all icons have the exact same type: ((props: SVGProps<SVGSVGElement>) => JSX.Element)
getNeededIcon(Beaker)

```

Reply:
Does it really matter if you can pass other icons? You'd be remove your `if` branches so it does not need to be a certain type anymore. If you really want to add type safety you can have an enum for the keys and a const for the tooltipIcons.